examples/ppo_trainer/run_deepseekv3_gsm8k_megatron.sh

```diff
@@ -0,0 +1,109 @@
+set -x
+
+
+# 0. download the model
+# only need to download the configuration_deepseek.py and config.json
+huggingface-cli download moonshotai/DeepSeek-V3-0324 configuration_deepseek.py config.json
```

Comment:
maybe add some comments to temp remove some keys like ``num_nextn_predict_layers`` and ``quantization_config`` in the config.json?